exporter/SynthesisFusionAddin/src/UI/ConfigCommand.py

```diff
@@ -231,111 +114,41 @@ def notify(self, args: adsk.core.CommandCreatedEventArgs) -> None:
         )
         apsSettings.tooltip = "Configuration settings for Autodesk Platform Services."
 
-        # clear all selections before instantiating handlers.
-        gm.ui.activeSelections.clear()
-
-        # ====================================== EVENT HANDLERS ======================================
-        """
-        Instantiating all the event handlers
-        """
-
-        onExecute = ConfigureCommandExecuteHandler()
-        cmd.execute.add(onExecute)
-        gm.handlers.append(onExecute)  # 0
-
-        onInputChanged = ConfigureCommandInputChanged(cmd)
-        cmd.inputChanged.add(onInputChanged)
-        gm.handlers.append(onInputChanged)  # 1
-
-        onExecutePreview = CommandExecutePreviewHandler(cmd)
-        cmd.executePreview.add(onExecutePreview)
-        gm.handlers.append(onExecutePreview)  # 2
-
-        onSelect = MySelectHandler(cmd)
-        cmd.select.add(onSelect)
-        gm.handlers.append(onSelect)  # 3
-
-        onPreSelect = MyPreSelectHandler(cmd)
-        cmd.preSelect.add(onPreSelect)
-        gm.handlers.append(onPreSelect)  # 4
-
-        onPreSelectEnd = MyPreselectEndHandler(cmd)
-        cmd.preSelectEnd.add(onPreSelectEnd)
-        gm.handlers.append(onPreSelectEnd)  # 5
-
-        onDestroy = MyCommandDestroyHandler()
-        cmd.destroy.add(onDestroy)
-        gm.handlers.append(onDestroy)  # 8
-
-
-class ConfigureCommandExecuteHandler(adsk.core.CommandEventHandler):
-    """### Called when Ok is pressed confirming the export
 
-    Process Steps:
-
-        1. Check for process open in explorer
-
-        1.5. Open file dialog to allow file location save
-            - Not always optimal if sending over socket for parse
-
-        2. Check Socket bind
-
-        3. Check Socket recv
-            - if true send data about file location in temp path
-
-        4. Parse file and focus on unity window
-
-    """
-
-    def __init__(self) -> None:
-        super().__init__()
-        self.current = SerialCommand()
+class ConfigureCommandExecuteHandler(PersistentEventHandler, adsk.core.CommandEventHandler):
+    """Called when the `Export` button is clicked from the main configuration panel."""
 
     @logFailure(messageBox=True)
-    def notify(self, args: adsk.core.CommandEventArgs) -> None:
+    def notify(self, _: adsk.core.CommandEventArgs) -> None:
+        design = adsk.fusion.Design.cast(adsk.core.Application.get().activeProduct)
         exporterOptions = ExporterOptions().readFromDesign()
 
-        if args.executeFailed:
-            logger.error("Could not execute configuration due to failure")
-            return
+        designNameSplit = design.rootComponent.name.split(" ")
+        if len(designNameSplit) > 1:
+            docName, docVersion = designNameSplit[:2]
```

Comment:
It looks like this line doesn't work properly with multi-word document names. When I try exporting the utility knife, `docName` becomes "Utility" and `docVersion` becomes "Knife".

Reply:
😬